src/main/resources/db/migration/postgres/R__check_for_standardizer_configuration_changes.java

```diff
@@ -0,0 +1,346 @@
+package com.labsynch.labseer.db.migration.postgres;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Date;
+
+import javax.transaction.Transactional;
+
+import org.flywaydb.core.api.migration.MigrationChecksumProvider;
+import org.flywaydb.core.api.migration.spring.SpringJdbcMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.dao.EmptyResultDataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+
+import com.labsynch.labseer.domain.StandardizationHistory;
+import com.labsynch.labseer.dto.configuration.MainConfigDTO;
+import com.labsynch.labseer.dto.configuration.StandardizerSettingsConfigDTO;
+import com.labsynch.labseer.utils.Configuration;
+
+public class R__check_for_standardizer_configuration_changes implements SpringJdbcMigration, MigrationChecksumProvider {
```

Comment:
@brianbolt I thought you had moved away from the repeatable flyway migration. Is it on purpose that this is still in here?

Reply:
Thanks, just removed it.